runtime/compiler/runtime/J9CodeCacheManager.cpp

```diff
@@ -385,6 +385,21 @@ J9::CodeCacheManager::allocateCodeCacheSegment(size_t segmentSize,
       }
 #endif
 
+#ifdef LINUX
+      if (_disclaimEnabled)
+         {
+         segmentType |= MEMORY_TYPE_VIRTUAL; // Make sure mmap is used for allocation
```

Comment:
I don't think this is needed. There are provisions in the VM to always allocate the code cache with mmap.

Reply:
Right, just wanted to confirm. Thanks!